mod/file/pages/file/view.php

```diff
@@ -36,7 +36,6 @@
 
 
 	$title = gc_explode_translation($file->title, $lang);
-	$file->title = gc_explode_translation($file->title, $lang);
```

Comment:
This change makes the breadcrumb on the file page have the json on the file name.